### ISSUE TITLE
Order lifeCycle Timestamp API

### ISSUE THREAD

**Issue by abdessamadb** [Sylius/Sylius#12691]
I can't find any way to get order created_at and updated_at i searched throw the serialization but now clue for that, I think this is missing from documentation or should be implemented 

`<property name="created_at" readable="true" writable="false"/>`

**Comment by SirDomin** [comment]
Yes, thats a good idea, feel free to open a PR with these values and contract tests for them :)

**Comment by NoResponseMate** [comment]
Hi @abdessamadb 👋 

We've added this to our backlog, will be most probably resolved in 1.14.
As for an immediate solution, you can always add these fields to the Order's serialization mapping as per [Symfony serializer docs](https://symfony.com/doc/6.4/components/serializer.html).

Cheers 🍻
